### ISSUE TITLE
cannot parse boolean in Postgresql

### ISSUE THREAD

**Issue by shusson** [taozhi8833998/node-sql-parser#363]
**Describe the bug**
Cannot parse a boolean type in postgres

**Database Engine**
postgres 12.1

**To Reproduce**

```sql
CREATE TABLE "foos"
(
    "Id" varchar(25) not null,
    "status" boolean null
);
```

```
Expected "--", "->", "->>", ".", "/*", "BIGINT", "CHAR", "DATE", "DATETIME", "DECIMAL", "DOUBLE", "FLOAT", "GEOMETRY", "INT", "INTEGER", "JSON", "JSONB", "LONGTEXT", "MEDIUMTEXT", "NUMERIC", "SMALLINT", "TEXT", "TIME", "TIMESTAMP", "TINYINT", "TINYTEXT", "UUID", "VARCHAR", or [ \t\n\r] but "b" found.
```

- node-sql-parser 2.2.2
- node 12.14

**Expected behavior**
Can parse boolean types. The workaround I have now is to replace boolean with tinyint(1).

**Comment by taozhi8833998** [comment]
got, I will add boolean type asap.

**Comment by taozhi8833998** [comment]
try `2.2.3` version

**Comment by shusson** [comment]
👍 fixed in 2.2.3
